css/styles.css

```diff
@@ -194,93 +194,41 @@
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
-  font-family: inter, serif;
+  font-family: 'Inter', serif;
   margin-right: 10px;
   margin-top: 10px;
 }
 
 .see-prjct-btn button {
-  background-color: #ff6b00;
+  background: #f55800;
   border: none;
   color: #fff;
   padding: 10px 10px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 16px;
-  font-family: 'Inter', serif;
+  font-family: "Inter", serif;
   margin-top: 10px;
+  font-weight: 700;
 }
 
 .see-prjct-btn button:hover {
   background-color: #fff;
-  border: 1px solid #ff6b00;
-  color: #ff6b00;
+  border: 1px solid #f55800;
+  color: #f55800;
```

Comment:
[OPTIONAL] Same issue as above :arrow_up:  . I can see you tried to fix the contrast but it wasn't enough!

- [ ] I suggest you increase the contrast by either changing the colors or by using [contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/contrast())